instructor/common.py

```diff
@@ -18,30 +18,41 @@
 
 
 @login_required
-def assignments(request):
+def assignments(request, error=''):
     assignments = models.Assignment.objects.filter(course__owner=request.user)
+    return render_to_response(
+        'instructor/assignments.html',
+        {
+            'error': json.dumps(error),
```

Comment:
Just adding this since I made this suggestion in the first place. Since the error is being used as a variable in Javascript wouldn't it make sense to JSON encode rather than use the string escaping in Django templates?